### PR TITLE
Initial PR for the application

### PR DESCRIPTION
**Pull request by trrenty**

### CODE REVIEW

File: application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/DefaultConfluenceMigratorManager.java [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+    private void addToJsonList(LogEvent logEvent, List<Map<String, Object>> logList)
+    {
+        Map<String, Object> logMap = new HashMap<>();
+        logMap.put("level", logEvent.getLevel().toString());
+        logMap.put("timeStamp", logEvent.getTimeStamp());
+        logMap.put("message", logEvent.getFormattedMessage());
+        if (logEvent.getThrowable() != null) {
+            logMap.put("throwable", ExceptionUtils.getStackFrames(logEvent.getThrowable()));
+        }
+        logList.add(logMap);
```
Comment (mflorea) [inline application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/DefaultConfluenceMigratorManager.java:193]: You are duplicating the information you have in the job status (serialized in the file system).

File: application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/DefaultConfluenceMigratorPrerequisites.java [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+ */
+@Component
+@Singleton
+public class DefaultConfluenceMigratorPrerequisites implements ConfluenceMigratorPrerequisites
+{
+    private static final String ERROR = "cross";
+
+    private static final String SUCCESS = "check";
+
+    private static final String WARNING = "exclamation";
```
Comment (mflorea) [inline application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/DefaultConfluenceMigratorPrerequisites.java:64]: Maybe use an Enum instead.

File: application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/SpaceQuestion.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+/**
+ * Space question.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+public class SpaceQuestion
+{
+    /**
+     * The map of spaces that can be imported from Confluence.
```
Comment (mflorea) [inline application-confluence-migrator-pro-api/src/main/java/com/xwiki/confluencepro/internal/SpaceQuestion.java:39]: Explain what the key and the value means.

**Comment by trrenty** [comment]
Merging the PR as it is to create a release candidate. The comments are to be honored in the close future.

**Pull request merged**